controllers/apim/api_controller.go

```diff
@@ -241,31 +274,57 @@ func rateLimitFiltersPatch(gateway client.ObjectKey) *istio.EnvoyFilter {
 		Patch: &prePatch,
 	}
 
-	postEnvoyFilterPatch := preEnvoyFilterPatch
-	postEnvoyFilterPatch.Patch = &postPatch
+	postAuthFilterPatch := preAuthFilterPatch
+	postAuthFilterPatch.Patch = postPatch
+
+	clusterPatch := networkingv1alpha3.EnvoyFilter_EnvoyConfigObjectPatch{
+		ApplyTo: networkingv1alpha3.EnvoyFilter_CLUSTER,
+		Match: &networkingv1alpha3.EnvoyFilter_EnvoyConfigObjectMatch{
+			Context: networkingv1alpha3.EnvoyFilter_GATEWAY,
+			ObjectTypes: &networkingv1alpha3.EnvoyFilter_EnvoyConfigObjectMatch_Cluster{
+				Cluster: &networkingv1alpha3.EnvoyFilter_ClusterMatch{
+					Name: "outbound|8081||limitador.kuadrant-system.svc.cluster.local",
```

Comment:
lets pull that out to a constant. Likely at some point it would come from configuration

Reply:
Done